packages/zipkin-transport-http/package.json

```diff
@@ -13,12 +13,15 @@
   "license": "Apache-2.0",
   "repository": "https://github.com/openzipkin/zipkin-js",
   "dependencies": {
+    "http-proxy-agent": "^2.1.0",
```

Comment:
what's the risk of having these dependencies?

Reply:
Do you mean security, amount of dependencies or compatibility?